README.md

```diff
@@ -60,6 +60,13 @@ Calls `aws cloudformation validate-template`
 
 Controls what cloudformation action to apply on the stack. This config is a required parameter. 
 
+### skip-deploy
+* **BitOps Property:** `skip-deploy`
+* **Environment Variable:** `CFN_SKIP_DEPLOY`
+* **default:** `none`
```

Comment:
Other empty vars are using `""`.
Why `none` for this one? Any difference between the behavior?

```suggestion
* **default:** `""`
```